File: examples/WorldPop_GP_100m_pop.js
var dataset = ee.ImageCollection("WorldPop/GP/100m/pop");

var visualization = {
  bands: ['population'],
  min: 0.0,
  max: 50.0,
  palette: ['24126c', '1fff4f', 'd4ff50']
};

Map.setCenter(113.643, 34.769, 7);

Map.addLayer(dataset, visualization, 'Population');

File: examples/WorldPop/WorldPop_GP_100m_pop.py
dataset = ee.ImageCollection('WorldPop/GP/100m/pop')

visualization = {
    'bands': ['population'],
    'min': 0.0,
    'max': 50.0,
    'palette': ['24126c', '1fff4f', 'd4ff50'],
}

m = geemap.Map()
m.set_center(113.643, 34.769, 7)
m.add_layer(dataset, visualization, 'Population')
m
